src/Symfony/Bundle/FrameworkBundle/Resources/config/validator_debug.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="debug.validator" decorates="validator" class="Symfony\Component\Validator\Validator\TraceableValidator">
```

Comment:
this service should be private, as you should never get it at runtime explicitly as `debug.validator`